src/app/settings/shelf/action-panel/action-panel-import/action-panel-import.component.ts

```diff
@@ -0,0 +1,69 @@
+import { Component, Input } from '@angular/core'
+import { MatSnackBar } from '@angular/material/snack-bar'
+import { ConfigurationService } from 'src/app/shared/services/configuration.service'
+import { ConfigurationParser } from 'src/app/shared/abstractions/configuration-parser.service'
+import { ShelfActionPanelConfiguration, ShelfActionPanelConfigurationWithId, ShelfConfiguration } from 'src/app/shared/models/shelf-configuration'
+
+@Component({
+  selector: 'sofie-action-panel-import',
+  templateUrl: './action-panel-import.component.html',
+  styleUrls: ['./action-panel-import.component.scss'],
+})
+export class ActionPanelImportComponent {
```

Comment:
This component duplicates a lot of the logic of `ActionTriggersImportComponent`.

Instead of duplicating the code we should have an import sevice that can load a text file and convert it to an object of a given type. Then these components can use that service for the actual logic of reading an parsing a file.